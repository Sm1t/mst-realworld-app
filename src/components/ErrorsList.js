import React from 'react';

const ErrorsList = ({ errors = [] }) => (
  <ul className="error-messages">
    {errors.map((error, index) => (
      <li key={index}>{error}</li>
    ))}
  </ul>
);

export default ErrorsList;
