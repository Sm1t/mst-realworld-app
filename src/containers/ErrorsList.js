import { observer } from 'mobx-react';

import ErrorsList from '../components/ErrorsList';

export default observer(ErrorsList);
