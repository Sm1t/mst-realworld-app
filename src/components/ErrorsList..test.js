import React from 'react';
import { shallow } from 'enzyme';

import ErrorsList from './ErrorsList';

describe('Login', () => {
  it('should render without problems', () => {
    const errorsList = shallow(<ErrorsList />);

    expect(errorsList).toMatchSnapshot();
  });
});
