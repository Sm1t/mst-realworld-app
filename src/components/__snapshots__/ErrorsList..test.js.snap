// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login should render without problems 1`] = `ShallowWrapper {}`;
